test/com/puppetlabs/http/client/impl/metrics_unit_test.clj

```diff
@@ -23,21 +23,24 @@
           (TimerUtils/startFullResponseTimers metric-registry
                                            (BasicHttpRequest. "GET" "http://localhost/foo")
                                            nil
-                                           Metrics/DEFAULT_NAMESPACE_PREFIX)
+                                           Metrics/DEFAULT_NAMESPACE_PREFIX
+                                              true)
           (is (= (set (list url-id url-method-id)) (set (keys (.getTimers metric-registry)))))))
       (testing "metric id timers are not created for a request with an empty metric id"
         (let [metric-registry (MetricRegistry.)]
           (TimerUtils/startFullResponseTimers metric-registry
                                            (BasicHttpRequest. "GET" "http://localhost/foo")
                                            (into-array String [])
-                                           Metrics/DEFAULT_NAMESPACE_PREFIX)
+                                           Metrics/DEFAULT_NAMESPACE_PREFIX
+                                              true)
```

Comment:
I realize this is entirely my fault, but I think the alignment here and at line 27 is wrong (for the whole function).